feat: Mention install discord-api-types in registering slash commands (#802)
discordjs/guide: guide/interactions/registering-slash-commands.md

> First off, install the [discord.js REST module](https://github.com/discordjs/discord.js-modules/blob/main/packages/rest/) by running `npm install @discordjs/rest` in your terminal.
> 
> Then, install the [Discord API Types](https://github.com/discordjs/discord-api-types) package by running `npm install discord-api-types` in your terminal.

Reviewer: ```suggestion
First off, install the [`@discord.js/rest`](https://github.com/discordjs/discord.js-modules/blob/main/packages/rest/) and [`discord-api-types`](https://github.com/discordjs/discord-api-types/) by running `npm install @discordjs/rest discord-api-types ` in your terminal.
```
This can be a single command, and thus a single sentence
